react-context/client/src/pages/AuthPage.tsx

```diff
@@ -0,0 +1,40 @@
+import { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { FaBolt } from 'react-icons/fa';
+import { AuthForm } from '../components/AuthForm';
+import { AppContext } from '../components/AppContext';
+
+type Props = {
+  action: 'sign-in' | 'sign-up';
+};
+export function AuthPage({ action }: Props) {
+  const appContext = useContext(AppContext);
```

Comment:
The nicest way to do this is with destructuring:
`const { user, handleSignIn } = useContext(AppContext);`